guides/v15-mdc-migration.md

```diff
@@ -446,9 +446,9 @@ TODO(mmalerba): link to density docs once they exist.
 
 ### Progress Bar
 
-* Hiding the bar with `visibility: hidden` will not hide all internal elements because they apply
-  a `visibility: visible` style. Instead, style it with `opacity: 0` or `display: none`, or remove
-  it completely with `ngIf`.
+* Visibility on internal elements is now set to `visible`. Setting `visibility: hidden` will no
+  longer hide all internal elements. Instead, style it with `opacity: 0`, `display: none`, or
+  remove it completely with `ngIf`.
```

Comment:
```suggestion
* You can no longer hide the progress-bar by applying `visibility: hidden` because 
  internal elements within the progress-bar apply `visibility: visible`. You can alternatively
  hide the progress with `opacity: 0`, `display: none`, or `ngIf`.
```
suggestion for passive voice, present tense